anonymous-message-board/tests/2_functional-tests.js

```diff
@@ -56,12 +58,10 @@ suite('Functional Tests', function() {
           assert.property(res.body[0], 'bumped_on');
           assert.property(res.body[0], 'text');
           assert.property(res.body[0], 'replies');
-          assert.property(res.body[0], 'reported');
-          assert.property(res.body[0], 'delete_password');
+          // assert.property(res.body[0], 'reported');
+          // assert.property(res.body[0], 'delete_password');
```

Comment:
These aren't actually sent back by the endpoint (presumably because they're private). Not sure if we want to send them back, or keep them hidden.

For now, removing the assertions pass the tests.

Reply:
Commenting these out is fine, but could you leave a comment in the code explaining why?  Not a huge deal, since we can always look at this PR, but it makes it less of a hassle in future.

Reply:
Re: https://github.com/freeCodeCamp/freeCodeCamp/pull/43041#issuecomment-926094009